website/src/categories/getting-started/developers.md

```diff
@@ -99,10 +123,42 @@ This will load the button component and register it as `<sl-button>`. After that
 
 ## Example project
 
-To serve as an inspiration (or just something to copy-paste from) we've made an example project that uses several Design System components so you can study how they interact or have something to compare your code to if something doesn't work as expected.
+To serve as an inspiration (or just something to copy-paste from) we've made some example projects that uses several Design System components so you can study how they interact or have something to compare your code to if something doesn't work as expected.
 
-You can find the [example project repository on Github](https://github.com/sl-design-system/example-design-system-lit-app)
+You can find the [Angular example project repository on Github](https://github.com/sl-design-system/angular-demo)
```

Comment:
Do we want to add the information (here in the documentation website) that CUSTOM_ELEMENTS_SCHEMA should be also added in the Angular or it is not necessary?